test/test.js

```diff
@@ -404,3 +404,55 @@ describe('List Schema Tests', function() {
+         const expected = "'''this ''is ++text++''''' ++''with'' styles++"
+
+         const output = serializeStandardTestCase(input)
+         assert.equal(expected, output)
+      })
+   })
+
+   describe('Interleaved bold and italic marks', function() {
+      it('should open and close correctly', function() {
+         const input = {"type":"doc","content":[{"type":"paragraph","content":[ { "type": "text", "marks": [{"type": "strong"}], "text": "this " }, { "type": "text", "marks": [{"type": "strong"},{"type": "em"}], "text": "is " }, { "type": "text", "marks": [{"type": "em"}], "text": "text " }, { "type": "text", "marks": [{"type": "strong"}], "text": "with styles" } ]}]}
+         const expected = "'''this ''is''''' ''text'' '''with styles'''"
+
+         const output = serializeStandardTestCase(input)
+         assert.equal(expected, output)
+      })
+   })
+
+   describe('Interleaved bold and italic marks without spaces ', function() {
+      it('should open and close correctly', function() {
+         const input = {"type":"doc","content":[{"type":"paragraph","content":[ { "type": "text", "marks": [{"type": "strong"}], "text": "this" }, { "type": "text", "marks": [{"type": "strong"},{"type": "em"}], "text": "is" }, { "type": "text", "marks": [{"type": "em"}], "text": "text" }, { "type": "text", "marks": [{"type": "strong"}], "text": "with styles" } ]}]}
+         const expected = "'''this''is'''''''text'''''with styles'''"
+
+         const output = serializeStandardTestCase(input)
+         assert.equal(expected, output)
+      })
+   })
+
+   describe('Formatted words with punctuation', function() {
+      it('should have their marks open and close correctly', function() {
+         const input = {"type":"doc","content":[{"type":"paragraph","content":[{"type":"text","marks":[],"text":"This is a "},{"type":"text","marks":[{"type":"strong"},{"type":"em"},{"type":"underline"}],"text":"thing!"},]}]}
+         const expected = "This is a '''''++thing!++'''''"
+
+         const output = serializeStandardTestCase(input)
+         assert.equal(expected, output)
+      })
+   })
+
+   describe('Adjacent links', function() {
+      it('should render correctly', function() {
+         const input = {"type":"doc","content":[{"type":"paragraph","content":[{"type":"text","marks":[{"type":"link","attrs":{"href":"http://example.com"}}],"text":"this link"},{"type":"text","marks":[{"type":"link","attrs":{"href":"https://www.ifixit.com"}}],"text":"is near this one"},]}]}
```

Comment:
Is it possible to have these be separated out like they were in the other file so they're more readable? Maybe trim out the extra space/lines before passing it to `serializeStandardTestCase`? That might be for another pull though

Reply:
I was kind of thinking about ways to make the test cases more readable, since most of it is kind of boilerplate-y JSON stuff.

It turns out the maintainer has another repo for building easily building documents for testing. It's probably worth making an issue to just redo most of the test cases in the more concise and easy to read format.